x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_loader.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingChart } from '@elastic/eui';
+import { AlertSummaryWidgetProps } from '..';
+
+type Props = Pick<AlertSummaryWidgetProps, 'fullSize'>;
+
+export const AlertSummaryWidgetLoader = ({ fullSize }: Props) => {
```

Comment:
Extracted to be used both on lazy loading and while loading data in the component